quinn-proto/src/endpoint.rs

```diff
@@ -748,17 +691,115 @@ impl fmt::Debug for Endpoint {
         fmt.debug_struct("Endpoint")
             .field("rng", &self.rng)
             .field("transmits", &self.transmits)
-            .field("connection_ids_initial", &self.connection_ids_initial)
-            .field("connection_ids", &self.connection_ids)
-            .field("connection_remotes", &self.connection_remotes)
-            .field("connection_reset_tokens", &self.connection_reset_tokens)
+            .field("index", &self.index)
             .field("connections", &self.connections)
             .field("config", &self.config)
             .field("server_config", &self.server_config)
             .finish()
     }
 }
 
+/// Maps packets to existing connections
+#[derive(Default, Debug)]
+struct ConnectionIndex {
```

Comment:
Not sure `Index` feels like the most obvious suffix here. Maybe `Map`?

Reply:
I think `Index` is descriptive here. A map owns its values; an index (as in database index) merely refers to them.